PowerUp-2018/src/com/spikes2212/robot/Commands/commandGroups/autonomousCommands/SideToScaleAuto.java

```diff
@@ -0,0 +1,55 @@
+package com.spikes2212.robot.Commands.commandGroups.autonomousCommands;
+
+import java.util.function.Supplier;
+
+import com.spikes2212.dashboard.ConstantHandler;
+import com.spikes2212.genericsubsystems.drivetrains.commands.DriveArcade;
+import com.spikes2212.robot.Robot;
+import com.spikes2212.robot.SubsystemComponents;
+import com.spikes2212.robot.SubsystemConstants;
+import com.spikes2212.robot.Commands.commandGroups.MoveLift;
+import com.spikes2212.robot.Commands.commandGroups.PlaceCube;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class SideToScaleAuto extends CommandGroup {
+	
+	public static final Supplier<Double> FORWARD_SPEED = ConstantHandler
+			.addConstantDouble("scale auto - forward speed", 0.4);
+	//TODO check rotation side
+	public static final Supplier<Double> ROTATION_SPEED = ConstantHandler
+			.addConstantDouble("scale auto - rotation speed", 0.6);
+	public static final Supplier<Double> ROTATION_TIME = ConstantHandler
+			.addConstantDouble("scale auto  - rotation time", 0.5);

```

Comment:
these rotation values are constant and useful to any 90 degrees rotation. for now - save them in the SubsystemConstants class.